django/contrib/admin/checks.py

```diff
@@ -465,8 +465,6 @@ def _check_ordering_item(self, obj, model, field_name, label):
         else:
             if field_name.startswith('-'):
                 field_name = field_name[1:]
-            if field_name == 'pk':
-                return []
```

Comment:
Ah nevermind, it doesn't matter. It's part of a check that only cares about M2M so no behavior change at all.